packages/psss/src/components/Table/CellComponents/AlertMenuCell.js

```diff
@@ -2,12 +2,30 @@
  * Tupaia
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
-import React from 'react';
+import PropTypes from 'prop-types';
+import React, { useCallback, useContext } from 'react';
 import MoreVertIcon from '@material-ui/icons/MoreVert';
 import { IconButton } from '@material-ui/core';
+import { AlertsPanelContext } from '../../../containers';
 
-export const AlertMenuCell = () => (
-  <IconButton>
-    <MoreVertIcon />
-  </IconButton>
-);
+export const AlertMenuCell = React.memo(({ organisationUnit, period, syndromeName }) => {
+  const { setIsOpen, setData } = useContext(AlertsPanelContext);
+  const data = { organisationUnit, period, syndromeName };
```

Comment:
I'm curious about whether there is a unique id or code for an alert? My understanding is that it is just a survey response but it might still be nice to have the survey response id on the front end.